### PR TITLE
add dev_mode and tools for debugging

### PR DESCRIPTION
**Pull request by claysauruswrecks**
- export-keypair command
- list-keys command

## Summary by CodeRabbit

- **New Features**
  - Introduced a `dev_mode` option to enable developer-specific commands.
  - Added commands to export key pairs and list SSH keys when `dev_mode` is enabled.

- **Improvements**
  - Updated import paths for better module organization and clarity.
  - Enhanced SSH key management with new methods to retrieve and filter SSHKeyPair instances.

### CODE REVIEW

File: agentdesk/key.py [hunk 1]
```diff
@@ -32,6 +32,16 @@ def __post_init__(self) -> None:
...
 
+    @classmethod
+    def all(cls) -> List["SSHKeyPair"]:
+        """
+        Retrieve all SSHKeyPair instances from the database.
+        """
+        for db in cls.get_db():
+            records = db.query(SSHKeyRecord).all()
+            return [cls.from_record(record) for record in records]
+        raise Exception("No database session available")
```
[the remark this answers is not in the document]
Comment (pbarker) [reply agentdesk/key.py:43]: `find` with no parameters will return all

**Review by pbarker — COMMENTED** [review]
TLAR would be fun to add an `ssh` command in the future

File: agentdesk/cli/main.py [hunk 2]
```diff
@@ -264,5 +277,65 @@ def clear_cache():
...
+            file.write(key.public_key)
+        os.chmod(
+            public_key_file_name, 0o644
+        )  # Set file mode to read/write for the owner, and read for others
+
+        print(
+            f"Decrypted private key for device '{key.name}' saved to {private_key_file_name}"
+        )
+        print(f"Public key for device '{key.name}' saved to {public_key_file_name}")
+
```
[the remark this answers is not in the document]
Comment (claysauruswrecks) [reply agentdesk/cli/main.py]: I tossed an NFO banner on any export, does that work?

[the remark this answers is not in the document]
Comment (claysauruswrecks) [reply agentdesk/cli/main.py]: It's dev_mode, no seatbelts loaded.

**Review by pbarker — APPROVED** [review]
😎

**Pull request merged**